common/djangoapps/util/password_policy_validators.py

```diff
@@ -13,39 +13,105 @@
 from django.conf import settings
 from django.core.exceptions import ValidationError
 from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import ungettext_lazy as ungettext
 from Levenshtein import distance
 from six import text_type
 
+from student.models import PasswordHistory
 
-def validate_password_strength(value):
+
+class SecurityPolicyError(ValidationError):
+    pass
+
+
+def password_min_length():
+    """
+    Returns minimum required length of a password.
+    Can be overridden by site configuration of PASSWORD_MIN_LENGTH.
+    """
+    min_length = getattr(settings, 'PASSWORD_MIN_LENGTH', None)
```

Comment:
Nit: this can be simplified to just `return getattr(settings, 'PASSWORD_MIN_LENGTH', 2)`

Reply:
I had thought that too, but looks like some of our config files set PASSWORD_MIN_LENGTH = None! So I did it this way to catch that too.

Reply:
🔥 :shipit: